pkg/smbios/info64_test.go

```diff
@@ -88,94 +88,124 @@ func Test64GetTablesByType(t *testing.T) {
 	}
 }
 
-func systemInfoTable() (*Table, error) {
-	info, err := setupMockData()
-	if err != nil {
-		return nil, err
+func defulatType1Table() *Table {
```

Comment:
shouldn't it be `defaultType1Table`?

Reply:
the same goes for the other occurrences of the same name

Reply:
Yes, it's a typo, thanks